README.md

```diff
@@ -1,2 +1,58 @@
 # collection_template
-A GitHub Template repo to use as the basis for future repos
+A GitHub Template repo to use as the basis for future repos. See [Creating a repository from a template](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template) to start your own collection. This README.md contains the sections we recommend you include in your collection README.md, and includes embedded comments to help you determine what each section should contain. Delete this top-level section in your collection repo.
+
+# Collection Title
+<!-- Add CI and code coverage badges. Sample included below. -->
+[![CI](https://github.com/ansible-collections/kubernetes/workflows/CI/badge.svg?event=push)](https://github.com/ansible-collections/kubernetes/actions) [![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/kubernetes)](https://codecov.io/gh/ansible-collections/kubernetes)
```

Comment:
Should this point to the kubernetes repo?

Reply:
How about something like
```suggestion
[![CI](https://github.com/ansible-collections/REPONAMEHERE/workflows/CI/badge.svg?event=push)](https://github.com/ansible-collections/REPONAMEHERE/actions) [![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/REPONAMEHERE)](https://codecov.io/gh/ansible-collections/REPONAMEHERE)
```